pyomo/contrib/gdpopt/iterate.py

```diff
@@ -89,6 +90,18 @@ def algorithm_should_terminate(solve_data, config):
         solve_data.results.solver.termination_condition = tc.maxIterations
         return True
 
+    # Check time limit
+    total_time_elapsed = default_timer() - solve_data.timing.total.start
+    if total_time_elapsed > config.time_limit:
+        config.logger.info(
+            'MindtPy unable to converge bounds '
```

Comment:
Is the solver used here always going to be MindtPy?

Reply:
Good catch! I will fix the message.

Reply:
Fixed in f0a20454e3f4a83e61d8cebec060662def0debc9